build.ps1

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env pwsh
```

Comment:
No copyright header

Reply:
added. Wasn't sure which should be on top. Let me know if I have it wrong.

Reply:
My testing indicates that the shebang has to be on the first line of the script.

Reply:
Yes, shebang must be first line.  This would be the exception to the copyright header rule being first.